modules/display.js

```diff
@@ -0,0 +1,109 @@
+  }
+
+  displayForm() {
+    this.list = document.getElementById('list');
+    this.addNew = document.getElementById('addnew');
+    this.contact = document.getElementById('contact');
+    this.bookSection = document.getElementById('books');
+    this.createBook = document.getElementById('createBook');
+    this.contactInfo = document.getElementById('contactinfo');
+    this.hr = document.getElementById('hr');
+    this.createBook.style.display = 'flex';
+    this.contact.style.color = 'black';
+    this.addNew.style.color = 'blue';
+    this.list.style.color = 'black';
+    this.bookSection.style.display = 'none';
+    this.contactInfo.style.display = 'none';
+    this.hr.style.display = 'none';
+  }
+
+  displayContact() {
+    this.list = document.getElementById('list');
+    this.addNew = document.getElementById('addnew');
+    this.contact = document.getElementById('contact');
+    this.bookSection = document.getElementById('books');
+    this.createBook = document.getElementById('createBook');
+    this.contactInfo = document.getElementById('contactinfo');
+    this.contact.style.color = 'blue';
+    this.addNew.style.color = 'black';
+    this.list.style.color = 'black';
+    this.contactInfo.style.display = 'flex';
+    this.createBook.style.display = 'none';
+    this.bookSection.style.display = 'none';
+  }
+
+  initialize() {
+    this.list.addEventListener('click', this.displayList);
+    this.addNew.addEventListener('click', this.displayForm);
+    this.contact.addEventListener('click', this.displayContact);
+    this.form.addEventListener('submit', this.callbackFunction.bind(this));
+    this.displayData();
+  }
+
+  displayData() {
+    const storedBooks = localStorage.getItem('books');
+    this.books = storedBooks ? JSON.parse(storedBooks) : [];
+    this.bookSection.innerHTML = '';
+
+    this.books.forEach((book, index) => {
+      const bookElement = document.createElement('div');
+      bookElement.classList.add('book');
+      bookElement.innerHTML = `
+          <p id="title">"${book.title}" by ${book.author}</p>
+          <button class='btn' onclick='bookManager.removeBook(${index})'>Remove</button>`;
+      this.bookSection.appendChild(bookElement);
+    });
+  }
+
+  callbackFunction(event) {
+    event.preventDefault();
+    const myFormData = new FormData(event.target);
+    let formDataObj = {};
+    myFormData.forEach((value, key) => {
+      formDataObj[key] = value;
+    });
+    this.id += 1;
+    formDataObj = { ...formDataObj, id: this.id };
+    this.books.push(formDataObj);
+    const storedBook = JSON.stringify(this.books);
+    localStorage.setItem('books', storedBook);
+    this.displayData();
+  }
+
+  removeBook(index) {
+    this.books.splice(index, 1);
+    const storedBook = JSON.stringify(this.books);
+    localStorage.setItem('books', storedBook);
+    this.displayData();
+  }
+
+  static initialize() {
+    const bookManager = new BookManager();
+    return bookManager;
+  }
```

Comment:
- [ ] Kindly make sure the `removeBook` method works as expected because now the user can not remove the book as expected. Kindly fix this issue.

  - ### YOUR APP BEHAVIOUR

  ![chrome-capture-2023-6-12](https://github.com/Wils-FOURR-speed/Awesome-Books-ES6/assets/87197412/74f312e2-b145-4993-81f8-a20bd15e5a17)